tests/integration/security/sds_ingress/ingress_test.go

```diff
@@ -223,15 +223,15 @@ func TestSingleMTLSGateway_CompoundSecretRotation(t *testing.T) {
 						PrivateKey: ingressutil.TLSClientKeyA,
 						Cert:       ingressutil.TLSClientCertA,
 					}
-					ingressutil.SendRequestOrFail(t, ing, host, credName[0], ingressutil.Mtls, tlsContext,
+					ingressutil.SendRequestOrFail(t, ing, host, credName[0], ingressutil.Mtls, tlsContext, false, /* QUIC */
```

Comment:
would prefer to make a special SendQUICRequestOrFail or something, to avoid changing all of these.

Reply:
Done.